docs/features/feature-guides/properties/overview.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Overview
+---
+
+import FeatureAvailability from '@site/src/components/FeatureAvailability';
+
+# About DataHub Structured Properties
+<FeatureAvailability/>
+
+**DataHub Structured Properties** provide a structured and validated approach to attaching metadata to your Data Assets, enabling organizations to enforce consistency and quality in metadata collection.
+
+<p align="center">
+  <img width="90%"  src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/properties/custom_and_structured_properties.png"/>
+</p>
+
+## What are Structured Properties?
+
+**Structured Properties** are a configurable feature in DataHub that allow metadata to be associated with Assets in a structured format. Each Structured Property has a specific type (e.g., Date, Integer, DataHub Asset, Text) and can include validation rules, such as restricting to specific allowed values or adhering to a pre-defined format.
+
+Structured Properties can be added to all DataHub Asset types, including Datasets, Dashboards, Pipelines, and more. This ensures flexibility in managing metadata across your entire data ecosystem.
+
+Structured Properties can also be used within [**DataHub Compliance Forms**](../compliance-forms/overview.md) to crowdsource documentation, annotation, and classification tasks. By incorporating Structured Properties into Compliance Forms, organizations can define specific metadata collection requirements and assign them to relevant stakeholders, such as data owners, stewards, and subject matter experts. This enables large-scale governance initiatives to be efficiently managed while maintaining accuracy and accountability.
```

Comment:
I'd rather reference Structured Properties from Compliance Forms docs, as opposed to reference Compliance FOrms docs from here.

Sure, structured properties can be used in compliance forms -- but so can all the other regular attributes (ownership, documentation, domains, glossary terms, etc). Would we update ALL of those feature guides to reference compliance forms? (Probably not, since we are still trying to validate the actual value of the experimental compliance forms feature).

Cleaner to avoid referencing compliance forms here